frontend/app/src/main/java/com/example/speechbuddy/ui/models/TextToSpeechUiState.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.speechbuddy.ui.models
+
+data class TextToSpeechUiState(
+    val isButtonEnabled: ButtonStatusType = ButtonStatusType.PLAY
```

Comment:
```suggestion
    val buttonStatus: ButtonStatusType = ButtonStatusType.PLAY
```

변수명 `is~~?`은 보통 Boolean Type에 써서 조금 더 직관적인 이름으로 바꾸면 좋을 것 같습니다!